Source/Reporting/Read/Reporting/TrainingCaseReportsForListing/TrainingCaseReportForListingEventProcessor.cs

```diff
@@ -136,5 +136,10 @@ public void Process(InvalidReportFromUnknownDataCollectorReceived @event, EventS
             };
             _caseReports.Insert(caseReport);
         }
+        [EventProcessor("717251e0-b982-4ca8-a129-44f97e661796")]
+        public void Process(TrainingReportConvertedToLive @event, EventSourceId caseReportId) 
+        {
+            _caseReports.Delete(_caseReports.GetById(caseReportId.Value));
```

Comment:
Her burde det vel være add (eller create eller hva det heter) i stedet for delete?

Reply:
Neida, blir riktig det her :)

VI har flere lese-modeller av live case reports (5) enn vi har av training case reports (1).

Event prosessorene som lytter på de **Converted** eventene i Read-prosjektet skal kun slette riktig case report fra sin read model collection, vi har derimot en policy som også lytter på samme eventet som da spytter ut et CaseReport/TrainingReportRecieved event som da resulterer i en add

Reply:
Merk at det kan tenkes at vi vil om-strukturere litt på Read-siden av case reports da den er litt rotete og har endel duplikasjon. Vi kunne gjort det lettere med å heller ha færre ReadModels og flere Queries, for eksempel.

Reply:
Ok! Supert. Da er denne ok for meg :D (Ser at jeg skulle kommentert på engelsk :P)